sql/core/src/test/scala/org/apache/spark/sql/SparkSessionBuilderSuite.scala

```diff
@@ -281,4 +281,23 @@ class SparkSessionBuilderSuite extends SparkFunSuite with BeforeAndAfterEach {
       ()
     }
   }
+
+  test("SPARK-33139: Test spark.sql.legacy.allowModifyActiveSession") {
```

Comment:
Shall we use a more meaningful test name here?

Reply:
how about 
"SPARK-33139: Test SparkSession.setActiveSession/clearActiveSession"